umbral/point.py

```diff
@@ -124,10 +119,7 @@ def to_bytes(self, is_compressed=True):
         if is_compressed is set to True.
         """
         affine_x, affine_y = self.to_affine()
-
-        # TODO: Get size of curve via OpenSSL
-        order = openssl._get_ec_order_by_curve_nid(self.curve.curve_nid)
-        key_size = backend._lib.BN_num_bytes(order)
+        key_size = get_field_order_size_in_bytes(self.curve)
```

Comment:
Good catch, man! I think it's better to call that `coord_size` or `field_size`, as in the `from_bytes()` method.